.circleci/postgres-13.0-alpine-postgis3/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/appropriate/docker-postgis/issues/157#issuecomment-562812857
+
+FROM alpine:latest
+
+FROM circleci/postgres:13.0-postgis
+
+ENV POSTGRES_PASSWORD="postgres"
```

Comment:
why do we use this dockerfile instead of direclty using `circleci/postgres:13.0-postgis`

Reply:
I'll be using this instead - `postgis/postgis:13-3.0-alpine` . I'm guessing alpine might be required to run some dependencies for the tests. I'll be keeping this dockerfile for now, incase it doesn't work with this one.

Reply:
None of the other images seem to be working because I'm not able to set the env var `POSTGRES_PASSWORD` correctly for them.

Reply:
There's a node server that's also started for the tests and I guess that's why we need `alpine` too.